geth/terraform/goerli.tf

```diff
@@ -0,0 +1,28 @@
+provider "google" {
+    credentials = "${file("INSERT_GOOGLE_CLOUD_CREDENTIAL_JSON")}"
```

Comment:
What directory does the file need to be placed in?

Reply:
updated the readme